### PR TITLE
Several new FAQs for -63-3, -63A, and -63B

### PR DESCRIPTION
**Pull request by jimfenton**

### CODE REVIEW

**Review by jricher — APPROVED** [review]

File: _63B/kbareset.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: Is it permissible to use sets of questions and answers stored by the subscriber for password reset?
+---
+Self-serve password reset requires authentication of the account owner (subscriber) in order to reset the password. Response to sets of knowledge-based questions is a form of knowledge-based authentication (KBA), and is not allowed to be used as described in [FAQ B07 above](#q-b07) and [SP 800-63-3 Section 4.3.1](https://pages.nist.gov/800-63-3/sp800-63-3.html#431-authenticators):
+
+> Knowledge-based authentication, where the claimant is prompted to answer questions that are presumably known only by the claimant, also does not constitute an acceptable secret for digital authentication.
+
+KBA for password reset would leave the account vulnerable to takeover.
+
+Alternative authenticators for password reset include lists of look-up secrets and out-of-band device authentication. See [NIST Special Publication 800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html) for more details for these authentication processes.
+
```
Comment (jricher) [inline _63B/kbareset.md:12]: I think we should add here the notion of re-proofing the user as a means of account recovery. Go through the proofing process, assign new authenticators, revoke old authenticators, go on with life. It's drastic and heavyweight, but it's secure and uses existing mechanisms and patterns.

File: _63B/passwordmanagers.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+The compromise of the master secret to a password vault would require all passwords in the vault to be recreated. However, many password managers today provide two-factor capability and are designed in a way that cloud password services are not able to access the vault, even if compromised. Password managers contain much information that is valuable to cyber criminals, making them high-value targets, so securing these vaults is essential.
+
+In SP 800-63B, NIST has not explicitly recommended the use of password managers, but recommends that verifiers permit the use of "paste" functionality so that the subscriber can use a password manager if desired. If using a password manager, subscribers should:
+
+* Choose a long, complex passphrase for the master password to the password manager and protect it from being stolen. A passphrase can be made sufficiently long to protect against attacks while still allowing memorization.
```
Comment (jricher) [inline _63B/passwordmanagers.md]: Just noticed: be careful with "complex" here because it sounds like character complexity requirements.

Comment (DTemo) [reply _63B/passwordmanagers.md]: Best to remove "complex" here to avoid any misunderstanding with prior text.

File: _63B/kbareset.md (hunk 1 above)
Comment (DTemo) [reply _63B/kbareset.md:12]: This is probably necessary. Additional text -- " If alternative authenticators are not available for password reset, then the subscriber's identity will need to be re-proofed in order to gain account access. See NIST SP 800-63B section 6.1.2.3 Replacement of a Lost Authentication Factor for additional details."

**Review by DTemo — APPROVED** [review]
Looks good to go.

**Pull request merged**